samples/bluetooth/fast_pair/locator_tag/src/fp_adv.c

```diff
@@ -259,6 +262,11 @@ static bool fp_adv_rpa_expired(struct bt_le_ext_adv *adv)
 			(k_uptime_delta(&uptime) / MSEC_PER_SEC));
 	}
 
+	if (!is_enabled) {
+		LOG_INF("Fast Pair: RPA rotation blocked as the fp_adv module is disabled");
+		return false;
+	}
```

Comment:
This is not needed as we still create advertising sets dynamically. We allocate it in the _enable function and release it in the _disable function. The newly allocated set will get the new RPA with the valid status, so we do not need to do this.

This callback will also never get called in the `!is_enabled` (disabled) state as the advertising set is deallocated.

Reply:
We could assert proper `is_enabled` state then I think